strato

```diff
@@ -124,6 +124,24 @@ function fetchLogs {
   python fetchlogs ${withdb_flag}
 }
 
+function setPassword {
```

Comment:
I think we should have both - "setPassword" (to **set** the password on node start) and "enterPassword" (or whatever the best name is - to **enter** same password after the machine was restarted and vault-wrapper lost the password from memory).
Password entered with enterPassword should be verified in vault-wrapper by trying to decrypt some value from database.

Reply:
I thought that may be both this cases are covered with same vault-wrapper endpoint (/strato/v2.3/password) so I tried this:
1. set password `password1` on fresh node (success);
2. ./strato --stop
3. ./strato --start
4. set **different** password `password2` on same node (**success**);

I assume 4 should fail because the password in (4) is different to the password set initially.

Reply:
That's a good point. Vault Wrapper will return an error if the password entered is incorrect, which it checks by trying to decrypt the message stored in Postgres. I'll add code to retry if the endpoint returns an error.